commands/volumes/volume-start.go

```diff
@@ -41,15 +44,36 @@ func startBricks(c transaction.TxnCtx) error {
 			c.Logger().WithFields(log.Fields{
 				"volume": volname,
 				"brick":  b.Hostname + ":" + b.Path,
-			}).Info("would start brick")
+			}).Info("Starting brick")
 
 			brickDaemon, err := brick.NewDaemon(vol.Name, b)
 			if err != nil {
 				return err
 			}
 
-			err = daemon.Start(brickDaemon, false)
+			retries := 0
+		RETRY:
+			err = daemon.Start(brickDaemon, true)
 			if err != nil {
+				// Retry iff brick failed to start because of port being in use.
+				if exiterr, ok := err.(*exec.ExitError); ok {
+					if status, ok := exiterr.Sys().(syscall.WaitStatus); ok {
```

Comment:
I tried changing things, and it doesn't work well during type conversions. For example, this conversion fails for me - `status, ok := exiterr.Sys().(unix.WaitStatus)`. The [documentation](https://golang.org/pkg/os/#ProcessState.Sys) states to use syscall.WaitStatus

Only errno constants be can interchangably used between `syscall` and `unix` packages. This is because the definitions of these constants in `unix` package calls `syscall` ones.